packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/registries/delete/delete.module.js

```diff
@@ -0,0 +1,21 @@
+import angular from 'angular';
+import ngTranslateAsyncLoader from '@ovh-ux/ng-translate-async-loader';
+import uiRouter from '@uirouter/angularjs';
+
+import routing from './delete.routing';
+import component from './delete.component';
+
+const moduleName = 'ovhManagerPciAiDashboardRegistriesDelete';
+
+angular
+  .module(moduleName, [
+    ngTranslateAsyncLoader,
+    'oui',
+    'ovh-api-services',
```

Comment:
**nitpick:** `ovh-api-services` is now deprecated. Can you switch the calls to `$http`?